self-tests.yml

```diff
@@ -100,27 +100,19 @@ variables:
       - func: get-project
       - func: set-up-mongodb
       - func: run-make
-        vars: { target: "set-var set-project-var load-smoke-data" }
-      - command: shell.exec
+        vars: { target: "set-var" }
+      - func: run-make
+        vars: { target: "set-project-var" }
+      - func: run-make
+        vars: { target: "load-smoke-data" }
+      - command: subprocess.exec
```

Comment:
I changed the run-make function to use an arg list rather than a command string which means that listing a string of multiple targets is interpreted as 1 target